spark-atlas-connector/src/main/scala/org/apache/spark/sql/kafka010/atlas/KafkaHarvester.scala

```diff
@@ -70,8 +70,8 @@ object KafkaHarvester extends AtlasEntityUtils with Logging {
       e => pDescription.append(e).append(" ")
     }
 
-    if(destTopic.isDefined) {
-      pDescription.append(") Topics written into( ").append(destTopic.get).append(" )")
+    if (targetTopic.isDefined) {
```

Comment:
Do we need to get more information like which Kakfa cluster/master this topic belongs to.

Reply:
We can extract some more information from producer parameter like we are doing for topic, but it is pretty limited to destination topic. For source topic we encounter the root issue again: we need to access KafkaSourceRDD, but it doesn't expose consumer parameters as field, as well as KafkaSourceRDD is not a case class.

IMO topic is only the information we can extract both source and sink safely. Otherwise custom patch will be needed again.